client/packages/system/src/Item/ListView/ListView.tsx

```diff
@@ -25,30 +30,33 @@ const ItemListComponent: FC = () => {
     initialSort: { key: 'name', dir: 'asc' },
     filterKey: 'codeOrName',
   });
+  // TODO this is not the right place for it, see comment in method
+  useInitUnitStore();
   const { data, isError, isLoading } = useItems();
   const pagination = { page, first, offset };
   const navigate = useNavigate();
   const t = useTranslation('catalogue');
-  const formatNumber = useFormatNumber();
-
-  type ItemWithStats = ItemRowFragment & {
-    stats: {
-      averageMonthlyConsumption?: number | null;
-      availableStockOnHand?: number | null;
-      availableMonthsOfStockOnHand?: number | null;
-    };
-  };
 
-  const columns = useColumns<ItemWithStats>(
+  const columns = useColumns<ItemsWithStatsFragment>(
     [
       'code',
       'name',
-      // getItemVariantInputColumn(),
+      {
+        key: 'packUnit',
+        label: 'label.pack-unit',
+        align: ColumnAlign.Right,
```

Comment:
> think there's an exception somewhere, but generally: right align for numbers, left for text

I would argue this is the exception

I think this is better on the right side, since the column is closely linked to the numbers, it helps understand what the numbers mean, even without the packUnit things. I think having it on the right side helps address the other point about KDD, the one about helping user with consistent and clear column names and column placement.

Reply:
agree - thnx